### PR TITLE
fix: #68 Better humanize string functionality

### PR DESCRIPTION
**Pull request by miquelbeltran**
### Description :memo:
- **Purpose**: Fixes issue #68 where classes like `Symbol` caused crashes
- **Approach**: Implements a custom humanize method, similar to the original one.

The method has two key advantages:

- Generates a String as a single line
- Goes only 1 level deep into object properties.

The advantage over using `JSON.stringify` is that we have better control over how the String is constructed.

We don't need to worry on recursive/self-referencing objects since only one level is explored.

The new method preserves a similar format to the original one, so it should not disrupt current users.

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)

**Updates**

- Removed dependency `object-to-human-string` (which has not been updated for 9 years!)
- Implemented humanize error method.
- Added more tests

### Test plan :test_tube:

- Unit tests.
- Should test with examples as well.

### Author to check :eyeglasses:
- [ ] Project and all contained modules builds successfully
- [ ] Self-/dev-tested 
- [ ] Unit/UI/Automation/Integration tests provided where applicable
- [ ] Code is written to standards
- [ ] Appropriate documentation written (code comments, internal docs)

### Reviewer to check :heavy_check_mark:
- [x] Project and all contained modules builds successfully
- [ ] Change has been dev-/reviewer-tested, where possible
- [x] Unit/UI/Automation/Integration tests provided where applicable
- [x] Code is written to standards
- [x] Appropriate documentation written (code comments, internal docs)

### CODE REVIEW

**Comment by miquelbeltran** [comment]
Note: To merge after 1.1.0 is released

File: package.json [hunk 1]
```diff
@@ -63,7 +63,6 @@
   "dependencies": {
     "@types/express": "^4.17.21",
     "debug": "^4.3.4",
-    "object-to-human-string": "0.0.3",
```
Comment (sumitramanga) [inline package.json:66]: Awesome drop 💪🏽 9 years is a long time!

**Review by sumitramanga — APPROVED** [review]
LGTM 🧊

**Comment by sumitramanga** [comment]
I'll test in the `develop` branch

**Review by TheRealAgentK — APPROVED** [review]
LGTM

**Comment by miquelbeltran** [comment]
Thanks! merging now

**Pull request merged**